### ISSUE TITLE
Web installer doesn't update RabbitMQ and postgresql custom passwords in airtime.conf

### ISSUE THREAD

**Issue by amavarick** [LibreTime/libretime#330]
Upon initial install and deleting the /etc/airtime/airtime.conf, the web installer does not update the RabbitMQ password in /etc/airtime/airtime.conf.

Steps to duplicate:  On step 2 of web installer:
If initial install, run the cli command then...
`sudo rabbitmqctl change_password <username> <newpassword>`
If initial or after deleting airtime.conf for existing instance:
Then select advanced in step 2/5 on gui and enter new password.
Complete gui setup and the page will go to: 
http://yourpage.domain/?config
the page will not display.

Looking at airtime.conf, the default password is still airtime.
```
[rabbitmq]
host = 127.0.0.1
port = 5672
user = airtime
password = airtime
vhost = /airtime
```
Upon changing the password= value to what I set above in cli, ?config works and services look good.

I assume the issue is between the gui and updating the airtime.conf file since as soon as you make the change manually on airtime.conf things work.

For sure this applies to Ubuntu 16.04.  Not sure if it applies to other OS and not sure if other non-default parameters are applying in this or other pages.

PS: Love this app and lots of thanks to the hours people have put into this!

**Comment by Robbt** [comment]
I think this is probably the same bug as I reported in #288 but this issue has a better description of the problem.

**Comment by amavarick** [comment]
So airtime.conf also doesn't update a custom postgresql password either...
Steps to reproduce:
Perform standard install of libretime:
```
sudo git clone https://github.com/libretime/libretime.git
cd /root/libretime
./install -iap
```
Before using the GUI web installer, alter the airtime postgresql user password:
```
#change postgresql password
sudo -u postgres psql
ALTER USER airtime PASSWORD 'testpassword';
\q
```
Then start the web installer.  Instead of the default 'airtime' password on the first step, enter your 'testpassword'.  The option will go green and allow you to proceed to the next page.  However, when you get to assigning the media folder, it will error out about DB connectivity.  If you look at /etc/airtime/airtime.conf it has the default password of 'airtime' and not what you configured.  Altering it after the fact and starting the 4 airtime services will work but screens don't appear properly as the creation of the media folder and the completion of the web installer did not complete properly.

****Altering title to include postgresql.****

**Comment by hairmare** [comment]
I usually work around not wanting to do the setup wizard by pre-creating a config directory with everything needed (ie. the config file and for <2.0.0-alpha.3 also the separate analyzer config). This allows skipping the whole setup process and just logging in, as far as i remember the db is create on the first run anyhow (I might have done that manually to bootstrap my production deploy, I can't quite remember).

Anyway, I think #245 also had the RabbitMQ bug and #288 might also be related. Maybe #215 has something to do with this as well. I created a new setup label and am adding all of these to it (#333 which I also consider related is already in it).

**Comment by Robbt** [comment]
Ok, so I didn't fix the second part of this. 
The issue appears to be that setupMusicDirectory calls - 
 Propel::init(CONFIG_PATH . "airtime-conf-production.php");
to connect, whereas the temporary config is what should be used to create this connection. Going to try to fix this and append it onto my last PR.

**Comment by Robbt** [comment]
Ok, I think I have a fix for this and I've been able to get the setup to write a custom password to postgresql but I think we have the password hardcoded elsewhere in the application startup (or linked to somewhere other than the config file) because it fails to work once I've gotten through the set-up and so I'll create a PR for this fix, but now I need to turn over rocks until I figure out what is causing libretime to fail to work with a password other than airtime.

**Comment by amavarick** [comment]
I believe the locations for the static passwords are located:
database section:
/etc/airtime/airtime.conf
database driver section:
/usr/share/airtime/php/airtime_mvc/build/build.properties
dsn section:
/usr/share/airtime/php/airtime_mvc/build/runtime-conf.xml

**Comment by Robbt** [comment]
Yeah the question at this point is, what is the best way to fix this with the least amount of work.
with the PR I put in /etc/airtime/airtime.conf is now updated based upon the web interface but I'm a little unclear what the easiest way to get the propel settings be pulled from /etc/airtime/airtime.conf vs. the places spread around. I created a new bug #356 to document that specific issue as it is different from just the installer.
